scripts/init-submodules-no-riscv-tools-nolog.sh

```diff
@@ -4,75 +4,71 @@
 set -e
 set -o pipefail
 
-SKIP_VALIDATE=false
+RDIR=$(git rev-parse --show-toplevel)
+
+# get helpful utilities
+source $RDIR/scripts/utils.sh
+
+common_setup
 
 function usage
 {
-    echo "Usage: $0 [--skip-validate]"
+    echo "Usage: $0 [--batch]"
     echo "Initialize Chipyard submodules and setup initial env.sh script."
     echo ""
-    echo "  --skip-validate    Skip prompt checking for tagged release"
+    echo "  --batch    Skip prompt checking for tagged release"
 }
 
+BATCH=false
 while test $# -gt 0
 do
    case "$1" in
-        --skip-validate)
-            SKIP_VALIDATE=true;
+        --batch)
+            BATCH=true;
             ;;
-        -h | -H | --help)
+        -h | -H | --help | help)
             usage
             exit 1
             ;;
-        *) echo "ERROR: bad argument $1"
+        *)
+            echo "ERROR: bad argument $1"
             usage
             exit 2
             ;;
     esac
     shift
 done
 
-# Check that git version is at least 1.7.8
-MYGIT=$(git --version)
```

Comment:
Are you removing this because Conda always installs the correct git? Even so, I think it's a good idea to leave this around.

Reply:
Sure. I removed it because of conda but I can keep this for now